src/acp/matcher.ts

```diff
@@ -808,9 +809,9 @@ export function removeCreator(matcher: Matcher): Matcher {
  * @since Not released yet.
  */
 export function getClientAll(matcher: Matcher): WebId[] {
-  return getIriAll(matcher, acp.client).filter(
-    (client: WebId) => client !== solid.PublicOidcClient
-  );
+  return getIriAll(matcher, acp.client)
+    .concat(getStringNoLocaleAll(matcher, acp.client))
+    .filter((client: WebId) => client !== solid.PublicOidcClient);
```

Comment:
```suggestion
  return getIriAll(matcher, acp.client)
    .filter((client: WebId) => client !== solid.PublicOidcClient)
    .concat(getStringNoLocaleAll(matcher, acp.client));
```
The non-IRI clients would never match the filter condition.